From 32e797544db668ccafbddc65e91c6fe1c79fb802 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Wed, 11 Feb 2015 12:32:56 -0800
Subject: [PATCH 5001/5003] glext.h: Add missing include of stddef.h for
 ptrdiff_t

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 8ea9fa778a04d1535f994e39dcfdae7ee1add37e)
---
 include/GL/glext.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/GL/glext.h b/include/GL/glext.h
index a3873a6..dded95b 100644
--- a/include/GL/glext.h
+++ b/include/GL/glext.h
@@ -4662,6 +4662,7 @@ GLAPI void APIENTRY glVertexBlendARB (GLint count);
 
 #ifndef GL_ARB_vertex_buffer_object
 #define GL_ARB_vertex_buffer_object 1
+#include <stddef.h>
 typedef ptrdiff_t GLsizeiptrARB;
 typedef ptrdiff_t GLintptrARB;
 #define GL_BUFFER_SIZE_ARB                0x8764
-- 
2.6.0

